### PR TITLE
chore: remove legacy CMO code and draft orders toggle (FPLA-2754)

### PR DESCRIPTION
**Pull request by joez-f**
### JIRA link ###

https://tools.hmcts.net/jira/browse/FPLA-2754

### Change description ###

Remove old draft CMO code e.g old events + handlers + providers, and draft orders toggle 

note: AgreedCMOUploadedContentProvider has been kept, it will be used in FPLA-2731

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by samuelzerozero — CHANGES_REQUESTED** [review]
all the part about "review-cmo" ReviewCMOController (and dependencies) is now gone also, the old flow can be removed

File: service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/orders/UploadDraftOrdersController.java [hunk 1]
```diff
@@ -112,7 +101,7 @@ public void handleSubmitted(@RequestBody CallbackRequest request) {
         CaseData caseDataBefore = getCaseDataBefore(request);
         CaseData caseData = getCaseData(request);
 
-        publishEvent(service.buildEventToPublish(caseData, caseDataBefore));
+        publishEvent(new DraftOrdersUploaded(caseData));
```
Comment (samuelzerozero) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/orders/UploadDraftOrdersController.java:104]: 👍 inlined from if (featureToggleService.isDraftOrdersEnabled()) {
            return new DraftOrdersUploaded(caseData);
        }

File: service/src/main/java/uk/gov/hmcts/reform/fpl/service/cmo/DraftOrderService.java [hunk 2]
```diff
@@ -254,34 +250,6 @@ public UUID updateCase(UploadDraftOrdersData eventData, List<Element<HearingBook
         return bundles;
     }
 
-    public UploadCMOEvent buildEventToPublish(CaseData caseData, CaseData caseDataBefore) {
```
Comment (samuelzerozero) [inline service/src/main/java/uk/gov/hmcts/reform/fpl/service/cmo/DraftOrderService.java:257]: 👍 inlined to controller

File: service/src/test/java/uk/gov/hmcts/reform/fpl/service/cmo/DraftOrderServiceTest.java [hunk 3]
```diff
@@ -152,24 +143,6 @@ void shouldBuildDynamicListsFromHearings() {
...
-        void shouldSetCmoAsDefaultHearingOrderKind() {
-            when(featureToggleService.isDraftOrdersEnabled()).thenReturn(false);
-
-            UploadDraftOrdersData eventData = service.getInitialData(CaseData.builder().build());
-
-            assertThat(eventData.getHearingOrderDraftKind()).containsExactly(HearingOrderKind.CMO);
-        }
-
-        @Test
-        void shouldNotSetDefaultHearingOrderKind() {
```
Comment (samuelzerozero) [inline service/src/test/java/uk/gov/hmcts/reform/fpl/service/cmo/DraftOrderServiceTest.java:166]: should this be removed?

File: service/src/test/java/uk/gov/hmcts/reform/fpl/service/email/content/cmo/DraftOrdersUploadedContentProviderTest.java [hunk 4]
```diff
@@ -1,135 +0,0 @@
...
-import java.util.stream.Stream;
-
-import static org.assertj.core.api.Assertions.assertThat;
-import static uk.gov.hmcts.reform.fpl.enums.HearingType.CASE_MANAGEMENT;
-import static uk.gov.hmcts.reform.fpl.enums.JudgeOrMagistrateTitle.HER_HONOUR_JUDGE;
-import static uk.gov.hmcts.reform.fpl.enums.JudgeOrMagistrateTitle.HIS_HONOUR_JUDGE;
-import static uk.gov.hmcts.reform.fpl.enums.JudgeOrMagistrateTitle.MAGISTRATES;
-import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.wrapElements;
-
-@ContextConfiguration(classes = {DraftOrdersUploadedContentProvider.class})
```
Comment (samuelzerozero) [inline service/src/test/java/uk/gov/hmcts/reform/fpl/service/email/content/cmo/DraftOrdersUploadedContentProviderTest.java:30]: is the right test class to remove? DraftOrdersUploadedContentProvider vs DraftCMOUploadedContentProvider

**Review by samuelzerozero — CHANGES_REQUESTED** [review]

File: service/src/test/java/uk/gov/hmcts/reform/fpl/service/email/content/cmo/DraftOrdersUploadedContentProviderTest.java (hunk 4 above)
Comment (joez-f) [reply service/src/test/java/uk/gov/hmcts/reform/fpl/service/email/content/cmo/DraftOrdersUploadedContentProviderTest.java:30]: hm oops I think you are right 🤦

File: service/src/test/java/uk/gov/hmcts/reform/fpl/service/cmo/DraftOrderServiceTest.java (hunk 3 above)
Comment (joez-f) [reply service/src/test/java/uk/gov/hmcts/reform/fpl/service/cmo/DraftOrderServiceTest.java:166]: re-added

**Review by samuelzerozero — APPROVED** [review]

**Pull request merged**